src/test/java/com/wikia/webdriver/testcases/specialpagestests/VideosPageTests.java

```diff
@@ -1,90 +1,103 @@
-   */
-  @Test(groups = {"VideosPage", "VideosPageTest_001", "Media"})
-  public void VideosPageTest_001_UIComponentsPresence() {
-    SpecialVideosPageObject specialVideos =
-        new WikiBasePageObject().openSpecialVideoPageMostRecent(wikiURL);
-
-    Assertion.assertTrue(specialVideos.isHeaderVisible(), "Header is not visible");
-    Assertion.assertTrue(specialVideos.isAddVideoButtonClickable(),
-                         "Add video button is not clickable");
-    Assertion.assertTrue(specialVideos.isNewestVideoVisible(), "Newest video is not visible");
-  }
-
-  /**
-   * Checks if a video can successfully be deleted from the Special:Videos page. Specifically, this
-   * test checks if, after the video has been deleted, its title shows up in the delete confirmation
-   * presented by Global Notifications. (Note: This test also adds a video beforehand to make sure
-   * running this test is sustainable).
-   */
-  @Execute(asUser = User.STAFF)
-  @Test(groups = {"VideosPage", "VideosPageTest_002", "Media"})
-  @RelatedIssue(issueID = "SUS-755")
-  public void VideosPageTest_002() {
-    SpecialVideosPageObject specialVideos = new SpecialVideosPageObject(driver);
-    YoutubeVideo video = YoutubeVideoProvider.getLatestVideoForQuery(VIDEO_QUERY);
-
-    specialVideos.addVideoViaAjax(video.getUrl());
-    specialVideos.isNewVideoAdded();
-
-    String addedVideoTitle = specialVideos.getNewestVideoTitle();
-    String addedVideoTitlePattern = addedVideoTitle;
-    if(addedVideoTitle.endsWith(SUFFIX_FOR_LONG_TITLE)){
-      addedVideoTitlePattern = addedVideoTitle.replace(SUFFIX_FOR_LONG_TITLE, "");
+    static final String VIDEO_QUERY = "truth";
+    public static final String SUFFIX_FOR_LONG_TITLE = " ...";
+
+    /**
+     * Verify UI elements on the Special:Videos page Logged-Out
+     */
+    @Test(groups = {"VideosPage", "VideosPageTest_001", "Media"})
+    public void VideosPageTest_001_UIComponentsPresence() {
+        SpecialVideosPageObject specialVideos =
+                new WikiBasePageObject().openSpecialVideoPageMostRecent(wikiURL);
+
+        Assertion.assertTrue(specialVideos.isHeaderVisible(), "Header is not visible");
+        Assertion.assertTrue(specialVideos.isAddVideoButtonClickable(),
+                "Add video button is not clickable");
+        Assertion.assertTrue(specialVideos.isNewestVideoVisible(), "Newest video is not visible");
+    }
+
+    /**
+     * Checks if a video can successfully be deleted from the Special:Videos page. Specifically, this
+     * test checks if, after the video has been deleted, its title shows up in the delete confirmation
+     * presented by Global Notifications. (Note: This test also adds a video beforehand to make sure
+     * running this test is sustainable).
+     */
+    @Execute(asUser = User.STAFF)
+    @Test(groups = {"VideosPage", "VideosPageTest_002", "Media"})
+    @RelatedIssue(issueID = "SUS-755")
+    public void VideosPageTest_002() {
```

Comment:
Since you are working on this tests now, could you add more descriptive name for tests? E.g. `VideosPageTest_001_UIComponentsPresence` is much better then `VideosPageTest_002`